packages/db/src/loaders/test/index.ts

```diff
@@ -119,7 +119,7 @@ const migrationConfig = Config.detect({
 });
 migrationConfig.network = "development";
 
-const db = new TruffleDB(config);
+const db = connect(config as any);
```

Comment:
In theory it should be a `TruffleConfig`, but since this is just in a test, I figured it's not important. Happy to make it more type-friendly if you think it's worth it.

Reply:
Nah it's not that important.